src/utils/dateFormatter.ts

```
@@ -0,0 +1,33 @@
+export function formatDate(dateString: string): string {
+  if (!dateString) {
+    throw new Error('Date string is required');
+  }
+  const date = new Date(dateString);
+  if (isNaN(date.getTime())) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)